### PR TITLE
Fix background images not being loaded

### LINKED ISSUE #72: background image not rendered
i try with a div that has background image，but not rendered.

The relevant line in the code
https://github.com/1904labs/dom-to-image-more/blob/904ef1b428599d316c5ac67e3401e3d7788fdb0d/src/dom-to-image-more.js#L856-L866

i change line 861 to `backgroundNode.style.getPropertyPriority('background')` and it works

### PR DESCRIPTION
**Pull request by YoshiWalsh**
Fixes 1904labs/dom-to-image-more#72 (thanks @yuyanan58 for the suggested fix)

Also adds support for background-image property. This is also extensible to any other CSS properties which we might want to support in future.

### CODE REVIEW

**Comment by IDisposable** [comment]
Thanks for the contribution, will test a bit this weekend and roll a new release.

File: src/dom-to-image-more.js [hunk 1]
```diff
@@ -887,18 +887,26 @@
...
+                    if(!value) {
+                        return Promise.resolve();
+                    }
+
+                    return inliner.inlineAll(value)
+                        .then(function (inlined) {
+                            backgroundNode.style.setProperty(
+                                propertyName,
+                                inlined,
+                                backgroundNode.style.getPropertyPriority(propertyName)
```
Comment (IDisposable) [inline src/dom-to-image-more.js:904]: this should be `value` that we computed on line 893

Comment (JoshuaWalsh) [reply src/dom-to-image-more.js]: The third argument for `CSSStyleDeclaration.setProperty()` is `priority`. The fact that we previously provided `background` (synonymous with `value` after my refactor) is the main cause of issue #72. If the `priority` argument contains anything other than `"important"`, `""`, or `undefined` then the `setProperty` call is a no-op. (Tested in both Firefox and Chrome.

I added support for `background-image` (and potentially other properties in future) as a bonus, but the priority fix is the main improvement of this PR.

I have updated the PR to make the code clearer.

**Review by IDisposable — APPROVED** [review]

File: src/dom-to-image-more.js (hunk 1 above)
Comment (IDisposable) [reply src/dom-to-image-more.js]: You're 100% correct, sorry. Will test this weekend

**Review by IDisposable — APPROVED** [review]

**Comment by JoshuaWalsh** [comment]
Hi @IDisposable , I don't want to nag but could you please ping me once you've pushed this to `npm`? I'd like to remove my own private dependency from my project. Thanks!

**Comment by JoshuaWalsh** [comment]
2.13.0 is working, thanks very much!

**Comment by IDisposable** [comment]
Thanks for getting back to me!

**Pull request merged**